packages/grafana-ui/src/components/Forms/Checkbox.tsx

```diff
@@ -14,10 +14,11 @@ export interface CheckboxProps extends Omit<HTMLProps<HTMLInputElement>, 'value'
   value?: boolean;
   // htmlValue allows to specify the input "value" attribute
   htmlValue?: string | number;
+  labelClassName?: string;
```

Comment:
Not enthusiastic about adding another class name prop in. 

Your usage appears pretty standard - is there a reason we need this class? Is there a bug in the Checkbox component that should be fixed instead?

Reply:
Yes, I agree. Maybe the best solution is to change the checkbox component itself. Basically, because of having the css       `white-space: nowrap`; and then a fixed `max-width: 480px`, the checkbox label is not showing well. You can check the images that I just updated .

Reply:
We should need to add this css in checkbox label
```
 max-width: fit-content;
 line-height: ${theme.typography.bodySmall.lineHeight};
 margin-bottom: 0;
```
Two of them are overriding the _labelStyles.label,_ attributes, which are max-width and margin-bottom.

Also remove 
`white-space: nowrap;`

This lets the label to have more than just one row and be contained inside the parent. It adds a line height in that case and the margin bottom should be 0 in order to be aligned with other components (I don't know why there's a margin-bottom for the label checkboxes)